### ISSUE TITLE
As a CIF user, I want to create a new revision (Amendment/General Revision/Minor Revision), so that I can filter, view, and update it via the holistic change log table

### ISSUE THREAD

**Issue by nanyangpro** [bcgov/cas-cif#923]
#### Description:
Prototype: https://www.figma.com/proto/PzOY8RyQnVXnjZJdSv2AqQ/Wireframing_Figma?page-id=2289%3A2663&node-id=3169%3A2715&viewport=-573%2C1082%2C0.31&scaling=scale-down&starting-point-node-id=3169%3A2715&show-proto-sidebar=1 (FYI the mocks are interactive)

As discussed with the team, the above workflow is agreed to be the only approach to make changes in the app, no matter what type of revision (Amendment/General Revision/Minor Revision) it is. The existing "Edit" button on each form should be discarded.

#### Acceptance Criteria:

Given I am a CIF administrator,
When I am on a form page for a project in 'view' mode,
Then the "Edit" button is no longer on the page.

Given I am  a CIF administrator,
When I click the "New Revision" button on the "Amendments and Other Revisions" page,
Then I am taken to the "New Revision" form page.

Given that I am a CIF administrator,
When I am on the "New Revision" page for a project,
Then there is a form requesting the: revision type, and which forms that will be updated.

Given that I am a CIF administrator,
When I click the "New Revision" button after filling in the form on the new revision page,
Then a new revision is created for the project, and I am navigated to the first form I selected in the "Forms to be Updated" field, in edit mode.

#### Development Checklist:

- [ ] The "edit" / "resume revision" button is no longer displayed on the project form pages.
- [ ] A "New Revision" button on the "Amendments and Other Revisions" page that takes users to the "New Revision" page. (The tasklist status for New Revision should be "In Progress" at this point)
- [ ] A new form for creating a new revision with the following fields:
- `revisionType` (radio buttons: "Amendment", "General Revision", "Minor Revision")
- Given "Amendment" is selected, a "Choose Type(s) for Amendment" field (Checkbox, `Schedule`, `Scope`, `Cost`)
- [ ] A "New Revision" button on the page that creates the new revision, and navigates to the first form in edit mode
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [x] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by mikevespi** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Climate-Action-Secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/923?planning-poker) with ZenHub @BCerki @dleard @gurjmatharu @pbastia @Sepehr-Sobhani

**Comment by pbastia** [comment]
A few questions need to be answered before we can implement this:
- What happens to the current revisions in the system? Should we make them all a "General Revision"?
- The "Forms to be updated" checkbox is still unclear how 
    a) the data entered will be stored, 
    b) how it impacts the rest of the revision (does it really constrain anything? or is it just an indication?)
    In any case, that third piece can be its own card with probably a lower priority, what do you think @LindsayMacfarlane @nanyangpro ?

**Comment by nanyangpro** [comment]
Hi @pbastia - As discussed,
- Yes.
- Let's make a separate card for the section and add "ux research" and "ux design" labels for further design iteration.

**Comment by dleard** [comment]
closed in favour of #992
